pacman/model/graphs/machine/destination_segmented_sdram_machine_partition.py

```diff
@@ -77,7 +78,7 @@ def get_sdram_base_address_for(self, vertex):
         for edge in self._edges:
             if edge.post_vertex == vertex:
                 return edge.sdram_base_address
-        return None
+        raise PacmanValueError(f"Vertex {vertex} is not in this partition")
```

Comment:
This is probably more useful than it was. 👍

Reply:
Yep Fail Fast unless there is a very good reason not to.